daemon/algod/api/algod.oas2.json

```diff
@@ -3880,6 +3884,23 @@
         }
       }
     },
+    "ScratchSlotWrite": {
```

Comment:
Can we just say "ScratchWrite" instead of "ScratchSlotWrite"?  It allows us to use "slot" as the field name and we don't have so much repetition.

Reply:
I think I like `ScratchChange` better than `ScratchWrite` or `ScratchSlotWrite`, for (in some probability) we might be interested in supporting backwards functionality, then we want to include old value in this struct, and `Change` feels better.

But in any case, we do want to remove `slot` part from the name.

Reply:
that's a good argument for "Change", I agree.

it's also a good argument for keeping the name "new-value". If we ever added to it, we'd call it old-value in this same object.